src/AmazonOrderList.php

```diff
@@ -67,7 +67,19 @@ public function __construct($s, $mock = false, $m = null)
         }
 
         if (isset($store[$s]) && array_key_exists('marketplaceId', $store[$s])) {
-            $this->options['MarketplaceId.Id.1'] = $store[$s]['marketplaceId'];
+            //Make possible to pass multiple MarketplaceIds
+            //The request will have a list of Marketplaces as:
+            //MarketplaceId.Id.x1 MarketplaceId.Id.x2 MarketplaceId.Id.x3
+            if (is_array($store[$s]['marketplaceId'])) {
+                $count_marketplace = count($store[$s]['marketplaceId']);
+                for ($i = 0; $i < $count_marketplace; ++$i) {
+                    $parameter_increment = $i + 1;
```

Comment:
You can just use a `foreach` loop here and initialise the `$parameter_increment` before the foreach loop with 1.

Reply:
Done